pycam/doas/ifit_worker.py

```diff
@@ -783,7 +798,8 @@ def _process_loop(self, continuous_save=True):
 
                 time_1 = time.time()
                 self.process_doas()
-                print('IFit Worker: Time taken to process {}: {}'.format(pathname, time.time() - time_1))
+                process_duration = time.time() - time_1
+                self.SpecLogger.info(f'Time taken to process {pathname}: {process_duration:.5f}')
```

Comment:
You can put simple arithmetic inside of f-strings, if process_duration isn't used elsewhere suggest that to avoid having another local variable floating around.
```suggestion
                self.SpecLogger.info(f'Time taken to process {pathname}: {time.time() - time_1:.5f}')
```